sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
+DOCS_BUILDER="doxygen"
+# we depend on these similar to putting doxygen[dot] in BDEPEND
+DOCS_DEPEND="media-gfx/graphviz media-libs/freetype"
+
+inherit autotools flag-o-matic fortran-2 python-r1 docs
+
+DESCRIPTION="A software package for jet finding in pp and e+e- collisions"
+HOMEPAGE="https://fastjet.fr/"
+SRC_URI="https://fastjet.fr/repo/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cgal doc examples python +plugins +contrib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# For now, we depend on cgal[gmp(-)] and append -lgmp to libs, despite possible incompatibility with cgal-5.5
+DEPEND="
+	contrib? ( sci-physics/fastjet-contrib )
+	cgal? ( sci-mathematics/cgal:=[shared(+),gmp(-)] )
+	plugins? ( sci-physics/siscone:= )
+	python? ( ${PYTHON_DEPS} )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	virtual/fortran
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-system-siscone.patch
+	"${FILESDIR}"/${P}-gfortran.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	use cgal && append-libs -lgmp
+	econf \
+		$(use_enable cgal) \
+		$(use_enable plugins allplugins) \
+		$(use_enable plugins allcxxplugins) \
```

Comment:
There is a small problem here. Because it seems that (some of) the plugins are being built anyway even with USE="-plugins". With the plugins flag disabled I get the following QA notice:
```
* QA Notice: Files built without respecting CFLAGS have been detected
*  Please include the following list of files in your report:
* /usr/lib64/libfastjetplugins.so.0.0.0

strip: x86_64-pc-linux-gnu-strip --strip-unneeded -N __gentoo_check_ldflags__ -R .comment -R .GCC.command.line-R .note.gnu.gold-version
/usr/lib64/libfastjetplugins.so.0.0.0
/usr/lib64/libfastjet.so.0.0.0
/usr/lib64/libfastjettools.so.0.0.0
```
This problem is also present in the in-tree version 3.0.6-r3 so it does not prevent merging this. Still it would be nice to fix this at some point.